tools/artillery/main.ts

```diff
@@ -87,7 +87,7 @@ async function runMachine(image: string, region: string): Promise<void> {
 async function main() {
   const build = await $`fly deploy --build-only --push --access-token=${FLY_API_KEY}`;
 
-  const imageRes = new RegExp("naming to (.+) done\n").exec(build.stdout.toString());
+  const imageRes = /naming to (.+) done\n/.exec(build.stdout.toString());
```

Comment:
Latest version of biome also added the enforcement of instantiating a RegExp class.

Reply:
weird, but sure